htcanalyze/main.py

```diff
@@ -21,15 +21,22 @@
 
 import configargparse
 
+# import own classes
 from htcanalyze.htcanalyze import HTCAnalyze, raise_value_error
 from htcanalyze.logvalidator import LogValidator
+# import for colorful output
 from rich import print as rprint, box
 from rich.progress import Table
 
-# typing identities
+
+# after analyzing, the information for each log file is saved
+# in it's on dictionary
 log_inf_list = List[dict]
+# dictionary with contents of all valid log files as a string
```

Comment:
Superfluous and wrong comment. This is a redeclaration of a variable type. It will never contain any values, as it is only used to provide a type declaration for function input and output (for example, `def foo(bar: List[dict])` is the same as `def foo(bar: log_inf_list)`.

Hence, I suggested in a previous comment not to use this, because it adds confusion.